ToDoManager.java

```diff
@@ -77,6 +78,22 @@ public static void executeUserCommand(String userCommand) {
 					System.out.println("Can not create new To Do. User must provide a description");
 				}
 				break;
+			case "complete":
```

Comment:
These "magic strings" for actions, as you know are not ideal. What's great about them is that this is a perfect opportunity to learn about enums in Java. You'll wish you had 'em in JS once you learn how they work.